infrastructure/platform.tfvars

```diff
@@ -20,34 +20,32 @@ project_id = "<your project ID>"
 ## network values
 create_network    = true
 network_name      = "ml-network"
-subnetwork_name   = "ml-subnet1"
+subnetwork_name   = "ml-subnet"
 subnetwork_cidr   = "10.100.0.0/16"
 subnetwork_region = "us-central1"
 
 ## gke variables
 private_cluster   = false ## true = private cluster, false = public cluster
 autopilot_cluster = false ## true = autopilot cluster, false = standard cluster
 cluster_name      = "ml-cluster"
-cluster_region    = "us-central1"
-cluster_zones     = ["us-central1-a", "us-central1-b", "us-central1-c"]
+cluster_location  = "us-central1-a"
 
 cpu_pools = [{
-  name           = "cpu-pool"
-  machine_type   = "n1-standard-16"
-  node_locations = "us-central1-b,us-central1-c"
-  autoscaling    = true
-  min_count      = 1
-  max_count      = 3
-  disk_size_gb   = 100
-  disk_type      = "pd-standard"
+  name         = "cpu-pool"
+  machine_type = "n1-standard-16"
+  autoscaling  = true
+  min_count    = 1
+  max_count    = 3
+  disk_size_gb = 100
+  disk_type    = "pd-standard"
 }]
 
 ## make sure required gpu quotas are available in that region
 enable_gpu = true
 gpu_pools = [{
   name               = "gpu-pool"
   machine_type       = "n1-standard-16"
-  node_locations     = "us-central1-a"
+  # node_locations = ["us-central1-a"] uncomment to target GPU node location to specified zones only
```

Comment:
sure we can do that.. 

This was commented so that people using other location for gke.. doesnot face issues with hard coded node_locations. With this intact, user need to make sure to update cluster_location and node_locations always.